client/webserver/site/src/js/settings.js

```diff
@@ -61,6 +63,27 @@ export default class SettingsPage extends BasePage {
     })
   }
 
+  async showDexInfo (page) {
+    const dexBox = page.dexBox
+    const dexInfo = page.dexInfo.cloneNode(true)
+    const dexInfoAcct = page.dexInfo.querySelector('.dexInfoAcct').cloneNode(true)
+
+    Doc.hide(dexBox)
+    page.dexInfo.remove()
+    dexInfo.removeAttribute('id')
+    dexInfo.querySelector('.dexInfoAcct').remove()
+
+    for (const [host, xc] of Object.entries(app.user.exchanges)) {
+      dexInfo.querySelector('.dexInfoHost').textContent = host
+      if (xc.isAuthed) {
+        dexInfoAcct.querySelector('.dexInfoAcctID').textContent = xc.acctID
```

Comment:
Instead of using `querySelector` and CSS classes, please use `data-tmpl` attributes and `Doc.tmplElement`. You can find examples all over.